tensilelite/Tensile/CustomKernels/Custom_Cijk_Alik_Bljk_BBS_BH_Bias_AS_SAV_UserArgs_MT256x256x64_MI16x16x1_SN_K1_MIWT4_16_DTVA.s

```diff
@@ -65,6 +65,8 @@ custom.config:
    1LDSBuffer: 1
    ScheduleIterAlg: 3
    DepthU: 64
+   StaggerU: 8

```

Comment:
Is this kernel only supports STU8 and WGM8?

Reply:
removed